dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -64,30 +47,40 @@ const mapAdTargeting = (adTargeting: AdTargeting): AdTargetParam[] => {
 // Variants for the Prebid server test
 // Assign each variant 4 groups e.g. 33.3% of content types each
 const variants = {
-	control: new Set([0, 1, 2, 3]),
-	'relevant-yield': new Set([4, 5, 6, 7]),
-	pubmatic: new Set([8, 9, 10, 11]),
+	'relevant-yield': new Set([0, 1, 4, 5, 6, 7]),
+	pubmatic: new Set([2, 3, 8, 9, 10, 11]),
```

Comment:
Here is where we re-allocate the groups that were originally in the control to the remaining variants.